tools/travis/build_tools.sh

```diff
@@ -7,5 +7,7 @@ if [ "$BUILD_TOOLS" = true ];
 then
     md5sum -c - <<< "49bc6b1b9ed56c83cceb6674bd97cb34 *html/changelogs/example.yml";
     cd tgui && source ~/.nvm/nvm.sh && gulp && cd ..;
+    php5.6 -l tools/WebhookProcessor/github_webhook_processor.php;
+    php5.6 -l tools/WebhookProcessor/secrets.php;
```

Comment:
You only need to run it on the actual processor don't you? secret.php is included by it

Reply:
yeah probably, I"m just focusing on getting php5.6 at all at this piont in time heh